measures/ApplyUpgrade/measure.rb

```diff
@@ -782,96 +763,81 @@ def get_hvac_system_values(hpxml_bldg, hvac_system_upgrades)
+      values['heating_system_2_heating_capacity'] = heating_system.heating_capacity
+      values['heating_system_2_heating_autosizing_factor'] = heating_system.heating_autosizing_factor
     end
 
     hpxml_bldg.cooling_systems.each do |cooling_system|
       next if hvac_system_upgrades.include?(cooling_system.id)
 
-      capacities['cooling_system_cooling_capacity'] = cooling_system.cooling_capacity
-      autosizing_factors['cooling_system_cooling_autosizing_factor'] = cooling_system.cooling_autosizing_factor
-      defect_ratios['cooling_system_airflow_defect_ratio'] = cooling_system.airflow_defect_ratio
-      defect_ratios['cooling_system_charge_defect_ratio'] = cooling_system.charge_defect_ratio
+      values['cooling_system_cooling_capacity'] = cooling_system.cooling_capacity
+      values['cooling_system_cooling_autosizing_factor'] = cooling_system.cooling_autosizing_factor
     end
 
     hpxml_bldg.heat_pumps.each do |heat_pump|
       next if hvac_system_upgrades.include?(heat_pump.id)
 
-      capacities['heat_pump_heating_capacity'] = heat_pump.heating_capacity
-      capacities['heat_pump_cooling_capacity'] = heat_pump.cooling_capacity
-      capacities['heat_pump_backup_heating_capacity'] = heat_pump.backup_heating_capacity
-      autosizing_factors['heat_pump_heating_autosizing_factor'] = heat_pump.heating_autosizing_factor
-      autosizing_factors['heat_pump_cooling_autosizing_factor'] = heat_pump.cooling_autosizing_factor
-      autosizing_factors['heat_pump_backup_heating_autosizing_factor'] = heat_pump.backup_heating_autosizing_factor
-      defect_ratios['heat_pump_airflow_defect_ratio'] = heat_pump.airflow_defect_ratio
-      defect_ratios['heat_pump_charge_defect_ratio'] = heat_pump.charge_defect_ratio
+      values['heat_pump_heating_capacity'] = heat_pump.heating_capacity
+      values['heat_pump_cooling_capacity'] = heat_pump.cooling_capacity
+      values['heat_pump_backup_heating_capacity'] = heat_pump.backup_heating_capacity
+      values['heat_pump_heating_autosizing_factor'] = heat_pump.heating_autosizing_factor
+      values['heat_pump_cooling_autosizing_factor'] = heat_pump.cooling_autosizing_factor
+      values['heat_pump_backup_heating_autosizing_factor'] = heat_pump.backup_heating_autosizing_factor
     end
 
-    return capacities, autosizing_factors, defect_ratios
+    return values
   end
 
   def get_panel_system_additions(args_hash)
     panel_system_additions = {}
     args_hash.each do |arg_name, _value|
-      if arg_name.start_with?('heating_system_') && (not arg_name.start_with?('heating_system_2_'))
+      if arg_name == 'hvac_heating_system'
         panel_system_additions['electric_panel_load_heating_system_new_load'] = true
-      elsif arg_name.start_with?('cooling_system_')
+      elsif arg_name == 'hvac_cooling_system'
         panel_system_additions['electric_panel_load_cooling_system_new_load'] = true
```

Comment:
@lixiliu @joseph-robertson If you have an upgrade from e.g. SEER 13 AC -> SEER 15 AC, it will set NewLoad=true for the AC on the upgraded home. It's not clear to me what impact that might have on the use of ResStock panel results. Is this something that should be addressed?

If we did want to address it, we either need more nuanced logic based on the detailed arguments or via inspection of the HPXML files.